Feat: AI plugins (#6779)
Kong/docs.konghq.com: app/_hub/kong-inc/ai-proxy/_changelog.md

## Changelog

### {{site.base_gateway}} 3.6.x

> * Introduced the *AI Proxy* plugin, which can mediate request and response formats, and authentication, between users and a number of supported Large Language Model AI services and models.

Author: Rephrasing this entry entirely, this whole sentence doesn't make sense.

> * Introduced the *AI Proxy* plugin, which can mediate request and response formats, and authentication, between users and a number of supported Large Language Model AI services and models.

Author: ```suggestion
* Introduced the AI Proxy plugin, which can mediate request and response formats, as well as authentication between users. This plugin supports a variety of Large Language Model (LLM) AI services.
```
